servicemigration/service.py

```diff
@@ -54,8 +56,9 @@ def __init__(self, name="", description="", startup="",
         self.name = name
         self.description = description
         self.startup = startup
-        self.endpoints = endpoints,
-        self.runs = runs,
-        self.volumes = volumes,
+        self.desiredState = desiredState
```

Comment:
Does Python have something equivalent to enumerated types? Would be good to add constants or the equivalent for the valid states rather than leaving it up to the SDK user to remember which int values go with which states